src/packet_distribution.cpp

```diff
@@ -540,12 +541,19 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     if (courtroom_constructed) {
       QVector<evi_type> f_evi_list;
 
-      for (QString f_string : f_contents) {
+      for (QString f_string : f_contents_encoded) {
         QStringList sub_contents = f_string.split("&");
 
         if (sub_contents.size() < 3)
           continue;
 
+        // decoding has to be done here instead of on reception
+        // because this packet uses & as a delimiter for some reason
+        sub_contents.replaceInStrings("<num>", "#")
+                    .replaceInStrings("<percent>", "%")
+                    .replaceInStrings("<dollar>", "$")
+                    .replaceInStrings("<and>", "&");
+
```

Comment:
Please use the existing function.
```suggestion
        AOPacket::escape(sub_contents);
```